scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 697,
```

Comment:
If the maximum bundle sized changed, then I suspect that so did the minimum and this value should be updated.